docs/Help.md

```diff
@@ -72,10 +72,10 @@ If you still can't see it, ask fourtf on [Chatterino Discord](https://discord.gg
 Log in to the [badge manager](https://manage.chatterino.com/) and click on *Disable My Chatterino Badge*.
 
 ### Login expired error
-If you're getting the `Login expired for user <user>! Try adding your account again.` error, simply re-add your account and it will start working again.
+If you're getting the `Login expired for user <user>! Try adding your account again.` error, simply re-add your account, and it will start working again.
```

Comment:
While I understand that the Oxford comma is preferred for some people, I still believe that a comma should only be used when the user is to half-pause mid-sentence. Which is not applicable in this scenario.

```suggestion
If you're getting the `Login expired for user <user>! Try adding your account again.` error, simply re-add your account and it will start working again.
```